database/insertData/03_organisations.js

```diff
@@ -65,115 +65,144 @@ exports.queries = [
       }`,
   // Associate users with organisations
   `mutation {
-        createUserOrganisation(
-          input: {
-            userOrganisation: { userId: 1, organisationId: 1, userRole: "Owner" }
-          }
-        ) {
-          user {
-            username
-          }
-          organisation {
-            name
-          }
+    createUserOrganisation(
+      input: {
+        userOrganisation: {
+          userId: 1
+          organisationId: 1
+          userRole: "Owner"
         }
-      }`,
+      }
+    ) {
+      user {
+        username
+      }
+      organisation {
+        name
+      }
+    }
+  }`,
   `mutation {
-        createUserOrganisation(
-          input: {
-            userOrganisation: { userId: 1, organisationId: 2 }
-          }
-        ) {
-          user {
-            username
-          }
-          organisation {
-            name
-          }
+    createUserOrganisation(
+      input: {
+        userOrganisation: {
+          #id: 2
```

Comment:
Would this comment be in? Want to do the same with the next userOrg ones?

Reply:
Okay, don't need it. This was leftover from when I was still thinking about putting the IDs in for these records as well, but it was getting unnecessarily complicated. Gone now :) af768c8

Reply:
Cool. Thanks!